x-pack/platform/plugins/shared/fleet/common/services/package_helpers.ts

```diff
@@ -42,3 +42,13 @@ export function getRootIntegrations(
     (pkg) => pkg!.name
   ).map((pkg) => ({ name: pkg!.name, title: pkg!.title }));
 }
+
+const INSTALL_SERVERS_INPUTS = ['cloudbeat', 'apm', 'fleet-server'];
+
+export function hasInstallServersInputs(packagePolicies: PackagePolicy[]): boolean {
+  return packagePolicies.some((policy) =>
+    policy.inputs.some(
+      (input) => INSTALL_SERVERS_INPUTS.includes(input.type) || input.type.startsWith('cloudbeat')
```

Comment:
included the `startsWith` condition, because cspm inputs are a prefix, e.g. `type: "cloudbeat/cis_eks"`